### PR TITLE
Additional project elements

### PR DESCRIPTION
**Pull request by mackyclemen**
This PR adds additional elements on the Project pages:

- cute little icons for each project
- AVP embed implementation

Will add more on this PR as time comes.

### CODE REVIEW

File: src/pages/projects/index.astro
Comment (mackyclemen) [inline src/pages/projects/index.astro:1]: indent changes are apparently recorded lmao

File: src/pages/projects/awards.astro
Comment (mackyclemen) [inline src/pages/projects/awards.astro:1]: how it currently looks atm:
image

**Review by karumadesu — APPROVED** [review]

**Comment by karumadesu** [comment]
Quite busy with stuff sorry but eto nalang siguro kulang and then it's already good for merging

- Page description
- Award description
- Maybe a bit of animations for consistency with other pages?
- Also, from where can we access the awards page HAHAHA

<details>
  <summary>do not open</summary>
  
  ```javascript
 penge naman kami award
  ```
  
</details>

**Comment by mackyclemen** [comment]
Should be ready.

_postnote: i can't undraft this - how_

**Review by karumadesu — APPROVED** [review]

**Pull request merged**